frontend/packages/kubevirt-plugin/integration-tests-cypress/tests/tier1-a/console.spec.ts

```diff
@@ -35,7 +35,7 @@ describe('Test VM console tab', () => {
     vm.start();
     cy.waitForLoginPrompt(YAML_VM_NAME, testName);
     tab.navigateToOverview();
-    cy.get(dashboardTab.guestAgentOK, { timeout: 300000 }).should('exist');
+    // cy.get(dashboardTab.guestAgentOK, { timeout: 300000 }).should('exist');
```

Comment:
Did you mean to leave this commented out?

Reply:
yes we had a problem with cirros and the guest agent is not in there by default, so commented it for now